### ISSUE TITLE
[Issue] bones with 'Rotate in Global Space" turned on make a keyframe automatically

### ISSUE THREAD

**Issue by PaulFinnerty** [JannisX11/blockbench#1720]
### Detailed description of your issue

version 4.6.0 Beta 1

If you have "Rotate in Global Space" turned on  for a bone in the timeline, the animation will create a keyframe automatically if you do not have one already, yet it does not show up in the timeline.

If intended this is very confusing, especially when you delete the last/only keyframe you had on the timeline to let the bone be like in bind pose again.

I suggest toggling on "Rotate in Global Space" alone does not create a keyframe by itself.

### Model format in which the issue occurs

Bedrock entity

### Blockbench variant

Program

### Operating System

Windows 10

**Comment by JannisX11** [comment]
In Minecraft, rotation relative to entity does not take effect unless there is a rotation keyframe with a notable value. That's why it has to generate a keyframe.

**Comment by PaulFinnerty** [comment]
Yes, I noticed that you put a value of 0.01 for one axis, because Minecraft does not apply the animation at all when it only has keyframes with 0,0,0 in the channels (even though it should in the case of relative to entity rotation).

That is not the issue for me, what I think is that toggling on "Rotate in Global Space" while you do not have a rotation keyframe yet in the timeline should not create one. Or at least, it should show up in the timeline, after all if you load the animation again, it will show up.
I can see why you'd want to make it hidden by default to prevent people from making the keyframe values 0,0,0 again, and get the bug in Minecraft; but it is still counter-intuitive in my opinion, that would be the only feature that works like that in Blockbench that I know.

Maybe the icon should only appear once you have at least one rotation keyframe in the timeline to be a little more clear? There is not much contrast between the greyed out icon and the activated icon on top of that, it was easy to forget about it. It could also be turned off automatically when you delete the last remaining rotation keyframes, hence not create this hidden keyframe?

https://user-images.githubusercontent.com/81629481/211174994-8bf4d9d2-7555-4bd1-8c6f-8bf1915f276d.mp4
